### ISSUE TITLE
XvY plots using line plot not useful

### ISSUE THREAD

**Issue by ggalibert** [aodn/imos-toolbox#546]
At the last QC summit @FelicityMc suggested XvY plots should be of a heatmap or scatter plot type for better readability.

**Comment by ocehugo** [comment]
I'm assuming this is a new option instead of removal of the previous one. Please advise otherwise.


Requirements:
- [ ] Implement pseudocolor XY map comparison
- [ ] Provide example image
- [ ] Evaluation/Tests in Linux/Osx/Windows

**Comment by ggalibert** [comment]
> I'm assuming this is a new option instead of removal of the previous one.

No, scatter plot type would replace current line plot type. I don't see any value in keeping a line plot type of graph.

**Comment by BecCowley** [comment]
@ocehugo, And, can we have the XvY plots available for the entire mooring? At the moment, the plot is only available for one instrument plus one other in the background. I'd like to see a version with all the instruments from the mooring (eg all SBE37 plotted on a T/S figure is very useful for deep water moorings)
